packages/issuer/src/blockchain-facade/CertificationRequest.ts

```diff
@@ -91,10 +92,14 @@ export class CertificationRequest extends PreciseProofEntity implements ICertifi
             events.find((log: BlockchainEvent) => log.event === 'CertificationRequested').topics[2]
         );
 
-        const success = await configuration.offChainDataSource.certificateClient.updateCertificationRequestData(
-            request.id,
-            { energy, files }
-        );
+        const success = await polly()
```

Comment:
What is the reason of polling here? Are we waiting for the backend to create the request based on the blockchain event?

Reply:
Yes

Reply:
Now I see that we can do it better. We can create a queue on the backend for these updates, so that when the backend reads the event, the queue is checked and entity updated.

Reply:
Yes, your proposal is ok, let's make it into separate PR. No need for that now.